showcase/src/app/demo-ui/components/table-with-custom-cell-rendering/table-with-custom-cell-rendering.component.html

```diff
@@ -0,0 +1,26 @@
+<stark-table [data]="data" [columnProperties]="columns" [filter]="filter">
+	<header><h1 class="mb0" translate>SHOWCASE.DEMO.TABLE.WITH_CUSTOM_CELL_RENDERING</h1></header>
+	<ng-container
+		[ngSwitch]="true"
+		*starkTableRowContent="
+			let implicitCtx = implicitCtx;
```

Comment:
This `implicitCtx` does not exist in Stark-Table. Could you please remove it since it's not useful ?
Also, could you please add some documentation on howto use `starkTableRowContent` directive ? 😊

Reply:
done